ci-operator/jobs/openshift/ironic-agent-image/openshift-ironic-agent-image-release-4.16-presubmits.yaml

```diff
@@ -475,6 +475,7 @@ presubmits:
         - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
         - --report-credentials-file=/etc/report/credentials
         - --target=[images]
+        - --target=[release:latest]
```

Comment:
Do we really need this? I also couldn't find a similar change when we moved to 4.16